<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select folder</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>26</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QListWidget" name="FileListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="DirectoryLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="SegmentationComboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="RemoveSmalRegionscheckBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove small regions</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="MinRegionSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="singleStep">
     <number>50</number>
    </property>
    <property name="value">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QComboBox" name="RawRegionMorfology1ComboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="RawRegionMorfology2ComboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="RawRegionMorfology3ComboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="RegionMorfology2ComboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="RegionMorfology1ComboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="RegionMorfology3ComboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="DivideRegionsCheckBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Divide regions</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowInputImageCheckBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show input image</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowThresholdedImgeCheckBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Thresholded image</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowWithoutHolesCheckBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show after hole removal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowAfter1MorphologyCheckBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show after first morphology</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FillHolesCheckBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>FillHoles</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="DurationLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="RegexLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="RegexListWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>211</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRegex">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowMaskCheckBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show mask</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowContourOnImageCheckBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Contour on image</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AllowResizeCheckBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>12</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Allow Image Resize</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RemoveBorderRegionCheckBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove regions touching border</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="FitEllipseCheckBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fit Ellipse</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RotateCheckBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowFitedElipseCheckBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Fited Rectangle</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowRotatedCheckBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Rotated</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CroppCheckBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>410</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cropp</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowCroppedCheckBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>200</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Cropped</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
